rclcpp/include/rclcpp/node_impl.hpp

```diff
@@ -66,9 +66,9 @@ Node::Node(const std::string & node_name, context::Context::SharedPtr context)
   using rclcpp::callback_group::CallbackGroupType;
   default_callback_group_ =
     create_callback_group(CallbackGroupType::MutuallyExclusive);
-  // TODO(esteve): remove hardcoded values
   events_publisher_ =
-    create_publisher<rcl_interfaces::msg::ParameterEvent>("parameter_events", 1000);
+    create_publisher<rcl_interfaces::msg::ParameterEvent>(
```

Comment:
Line 70 and 71 having the same indentation is kind of confusing. Can you remove the newline between line 69 and 70 and still be compliant with uncrustify?